test/integration/smoke-testing/test_footer_links.js

```diff
@@ -281,7 +281,7 @@ tap.test('clickScratchDayLink', options, t => {
 // SCRATCH CONFERENCE
 tap.test('clickScratchConferenceLink', options, t => {
     const linkText = 'Scratch Conference';
-    const expectedHref = '/conference';
+    const expectedHref = '/conference/2019';
```

Comment:
This will work fine until next year when we switch it to 2020. Would it be ok to use `t.match` instead of `t.equal` and then just make sure the url contains `'/conference'` or `'/conference/20'` if you want to check that the redirect happened.